lib/harmony/harmonydb/sql/20230712.sql

```diff
@@ -0,0 +1,41 @@
+create table sectorlocation
```

Comment:
Can we either make this `SectorLocation`, or the one below `storagelocation`, so that we're consistent with text case?

Reply:
May also want to pick slightly different names, `storagelocation` / `sectorlocation` look very similar visually, maybe `storage_bucket` / `storage_objects` (or `path`/`files`, just something that looks visually distinct)